g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/LocaleDataUtils.java

```diff
@@ -159,6 +159,38 @@ public String getDefaultRegionCode(String inputLocale, Map<String, String> regio
 		return defaultRegionMap;
 	}
 
+	/**
+	 * Get territory of a locale
+	 *
+	 * @param localeStr
+	 * @return
+	 */
+	public String getTerritory(String localeStr){
+		Locale locale = Locale.forLanguageTag(localeStr);
+		String territory = locale.getCountry();
+		if(territory == null || territory.isEmpty())//this is for supplement default region for locales like en, zh-Hans..
+			territory = getLanguageDefaultRegionMappings().get(localeStr);
```

Comment:
I remember there is an existing method to get the default region of a locale. Could you check?

Reply:
Confirmed with @chaoq0318, there is no such method.